catalyst/utils/data.py

```diff
@@ -82,9 +82,8 @@ def get_loaders_from_params(
     per_gpu_scaling: bool = False,
```

Comment:
`get_loaders_from_datasets` then?

Reply:
In this case, we should rename the `loaders_params` parameter to `loaders` too. Is it ok?

Reply:
we could not create loaders from... loaders 😂

Reply:
do you have a future vision for this function? what API should it have?

Reply:
```yaml
dataloaders:
  # loader params shared with all loaders
  batch_size: 32
  num_workers: 1
  datasets:
    train:
      _target_: MNIST
      train: true
      transform:
        _target_: A.Compose
        transforms:
          - *hard
          - *simple
    valid:
      train: y
      transform: *simple
  samplers:
    _key_value: true
    class_balance:
      _target_: ClassBalanceSampler
      _dataset_: train
  loaders:
    train:
      _dataset_: train
      _sampler_: class_balance
      num_workers: 1  # overwrite line 4
    valid:
      dataset: valid
```